data.tf

```diff
@@ -0,0 +1,22 @@
+data "aws_vpc" "vpc" {
+  id = var.vpc_id
```

Comment:
we are directly passing the vpc_id into var.vpc_id right, So why we are using data block here to get the same vcp_id again? via data block. Please correct me if I am wrong.